aea/cli/local_registry_sync.py

```diff
@@ -111,8 +108,9 @@ def enlist_packages(
     Generate list of the packages in local repo directory.
 
     :param base_dir: path or str of the local repo.
+    :param skip_consistency_check: whether or not to skip consistency checks.
 
-    :return: generator with Tuple of package_id and package directory.
+    :yield: generator with Tuple of package_id and package directory.
```

Comment:
The function "yields" a tuple, not a generator; thoughts?

```suggestion
    :yield: a tuple of package_id and package directory.
```